doc/api/worker_threads.md

```diff
@@ -296,7 +297,7 @@ if (isMainThread) {
     console.log('received:', value);
   });
 } else {
-  require('worker_threads').once('message', (value) => {
+  worker_threads.once('message', (value) => {
```

Comment:
This should instead use `worker_threads.parentPort` (see #21486). Might be helpful to apply that first.